pkg/polaris/graphql/aws/queries/all_aws_cloud_accounts_with_features.graphql

```diff
@@ -1,19 +1,36 @@
-query RubrikPolarisSDKRequest($feature: CloudAccountFeature!, $columnSearchFilter: String!) {
-    result: allAwsCloudAccountsWithFeatures(awsCloudAccountsArg: {columnSearchFilter: $columnSearchFilter, statusFilters: [], feature: $feature}) {
-        awsCloudAccount {
-            cloudType
-            id
-            nativeId
-            message
-            accountName
-        }
-        featureDetails {
-            feature
-            permissionsGroups
-            roleArn
-            stackArn
-            status
-            awsRegions
+query RubrikPolarisSDKRequest(
+  $feature: CloudAccountFeature!
+  $columnSearchFilter: String!
+  $statusFilters: [CloudAccountStatus]!
+) {
+  result: allAwsCloudAccountsWithFeatures(
+    awsCloudAccountsArg: {
+      columnSearchFilter: $columnSearchFilter
+      statusFilters: $statusFilters
+      feature: $feature
+    }
+  ) {
+    awsCloudAccount {
+      cloudType
+      id
+      nativeId
+      message
+      accountName
+    }
+    featureDetails {
+      feature
+      permissionsGroups
+      roleArn
+      stackArn
+      status
+      awsRegions
+      roleChainingDetails
+      mappedAccounts {
+        account {
+          id
+          name
         }
+      }
```

Comment:
I think this is what you want:
```
roleChainingDetails {
    roleArn
    roleUrl
}
mappedAccounts {
    account {
        id
        accountName
    }
}
```
Note, you would need to update the `aws.CloudAccountWithFeatures` type for the new fields to be unmarshaled. You might also need to add the new fields to the AWS cloud account type.